### PR TITLE
feat: update atlaskit package preset for changelog retrieval

### PR DESCRIPTION
**Pull request by setchy**
## Changes

Update atlaskit package rule to include `sourceUrl` and `sourceDirectory` for changelog retrieval 

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [x] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by viceice — CHANGES_REQUESTED** [review]
please update PR description to include reasons why this needs to be done.

this fundamentally changes the rule behavior

**Comment by setchy** [comment]
> please update PR description to include reasons why this needs to be done.

updated description now that https://github.com/renovatebot/renovate/pull/32701 landed

File: lib/config/presets/internal/packages.ts [hunk 1]
```diff
@@ -19,6 +19,8 @@ export const presets: Record<string, Preset> = {
   atlaskit: {
     description: 'All @atlaskit packages published by Atlassian.',
     matchPackageNames: ['@atlaskit/**'],
+    sourceDirectory: "design-system/{{ lookup (split packageName '/') 1 }}",
+    sourceUrl: 'https://bitbucket.org/atlassian/atlassian-frontend-mirror',
```
Comment (rarkins) [inline lib/config/presets/internal/packages.ts:23]: I don't think we _set_ anything in any of these packages: presets. Until now they're purely "matching" rules which only work if extended

Comment (setchy) [reply lib/config/presets/internal/packages.ts:23]: Thanks for the feedback

**Pull request closed without merging**